platform/components/insights/clusters/clusters.tsx

```diff
@@ -56,7 +56,7 @@ const Clusters: React.FC = () => {
     },
   );
   const totalNbTasks: number | null | undefined =
-    totalNbTasksData?.total_nb_tasks;
+    totalNbTasksData?.total_nb_tasks ?? 0;
```

Comment:
is this required ? Can't you handle the edge case where totalNbTasks is null or undefined in the other component ? 
This will cause visual glitches when rendering the first time, because the value will be set to zero and the error message "not enough messages" will be shown